### DISCUSSION TITLE
How to change current Github Campus to a new one?

### DISCUSSION THREAD

**Discussion by Code-Victor** [community/community#64857] [GitHub Education]
### Hi everyone,

I changed my university, and I'd like to update my school because I plan to apply for the GitHub campus expert program for my current school. I've tried reverifying my academic affiliations but it doesn't allow me to submit the form because of this👇
<img width="493" alt="image" src="https://github.com/community/community/assets/63613078/26eee5f0-6073-4442-896c-76496ca6c310">

any help? Thanks

**Comment by arafajs** [comment]
**You cannot** change your current GitHub Campus to a new one. GitHub Campus is a program that partners with educational institutions to provide students with benefits such as the GitHub Student Developer Pack. Once you are enrolled in a GitHub Campus program, you cannot switch to a different one.

**Reply by Code-Victor** [reply]
Thank you for your explanation. is it possible for me to apply for my current GitHub campus expert then without updating my school?

**Reply by arafajs** [reply]
@Code-Victor 
No, you cannot apply for GitHub Campus Expert without updating your school. One of the eligibility criteria for GitHub Campus Expert is that you must be enrolled in a formal higher education institution. If you are no longer enrolled in the institution that you listed on your application, you will need to update your school before you can apply.

**Reply by Code-Victor** [reply]
I guess that is a no-go for me then. Thanks for your help @arafajs

**Reply by arafajs** [reply]
@Code-Victor 
You're welcome.

**Comment by mishba-ai** [comment]
@Code-Victor Hey, I am also in the same boat as you were
Did you apply for the GitHub campus expert?

**Reply by Code-Victor** [reply]
No I haven't applied yet. But I plan to, hopefully it works despite this issue

**Reply by mishba-ai** [reply]
someone said to me to make the new account on GitHub and enroll for then student developer pack

**Reply by Code-Victor** [reply]
That might work but I think the requirement says you need a minimum of 6 months on GitHub before applying

**Reply by mishba-ai** [reply]
hey, did you fill out the GitHub global Campus form? have you created the new account?

**Comment by mishba-ai** [comment]
like In my case I want to update the email ID and the college for the student developer pack and also want to apply for GitHub campus expert 
can I apply for GitHub campus expert without updating the email address of my previous uni and ex-university name
plz help me with this

**Reply by Code-Victor** [reply]
@arafajs

**Comment by mishba-ai** [comment]
or do I have to again enroll for the student developer pack?

**Comment by lab596** [comment]
Hey, I'm not sure if this works but it is worth a try: 

If you go to GitHub settings and go to `Plans and usage` under Billing and Plans you are able to downgrade your GitHub Pro by clicking the edit button.

If this updates, you should be able to reapply for the GitHub student developer Pack with your new information

**Reply by 1u2hang** [reply]
I downgraded but it didn't work :(
